odoo/tools/misc.py

```diff
@@ -647,7 +673,23 @@ def posix_to_ldml(fmt, locale):
 
     return ''.join(buf)
 
-def split_every(n, iterable, piece_maker=tuple):
+
+@typing.overload
```

Comment:
I think you are confusing it with js, that's where we put useless stuff.

Reply:
It is not kind to the js team.

Just making typing information relevant for type inference. It's mainly for things like `split_every(100, records, records.browse)`. In any dynamic language, some functions are very generic and I find typing information serves as documentation.